doris/ddl_generic.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE bluesky (
+    kind VARCHAR(100) GENERATED ALWAYS AS (get_json_string(data, '$.kind')) NOT NULL,
+    operation VARCHAR(100) GENERATED ALWAYS AS (get_json_string(data, '$.commit.operation')) NULL,
+    collection VARCHAR(100) GENERATED ALWAYS AS (get_json_string(data, '$.commit.collection')) NULL,
+    did VARCHAR(100) GENERATED ALWAYS AS (get_json_string(data,'$.did')) NOT NULL,
+    time_us BIGINT GENERATED ALWAYS AS (get_json_bigint(data, '$.time_us')) NOT NULL,
+    `data` variant NOT NULL
+)
+DUPLICATE KEY (kind, operation, collection)
```

Comment:
Is the stuff in l. 9-13 mandatory or can we do without it?

Starrocks (starrocks/ddl_materialized.sql) uses a different primary key. For better comparability, it would be nice to use the same key, no?

Reply:
For doris's prefix index struct, the most important thing is to use several commonly used columns as key prefixes, and other key columns will hardly work.
In my test, using the `kind` column as the first key column can achieve relatively ideal performance test results, and other key columns are not important.